lib/features/upload/presentation/model/upload_file_state.dart

```diff
@@ -48,7 +45,6 @@ class UploadFileState with EquatableMixin {
       uploadingProgress: uploadingProgress ?? this.uploadingProgress,
       attachment: attachment ?? this.attachment,
       cancelToken: cancelToken ?? this.cancelToken,
-      fromFileShared: fromFileShared ?? this.fromFileShared
```

Comment:
why delete this flag?

Reply:
This flag is not required. It tells us that we need to call the `focus` editor when inserting an image. But now we have the `editor.hasFocus()` method (I published it in previous PRs). We can determine if the editor has `focus` and call the `requestFocus` method for it. You can watch the demo to see that the image sharing flow still works well for this update.